CONTRIBUTING.md

```diff
@@ -484,13 +484,15 @@ The oldest supported release (`YYMM`) can be found using
 nix-instantiate --eval -A lib.trivial.oldestSupportedRelease
 ```
 
-The release branches should generally only receive backwards-compatible changes, both for the Nix expressions and derivations.
+Users should be confident tracking the stable release channels for updates worry-free from breaking changes that will affect them.
+The release branches should only receive backwards-compatible changes, both for the Nix expressions and derivations.
+
 Here are some examples of backwards-compatible changes that are okay to backport:
 - ✔️ New packages, modules and functions
 - ✔️ Security fixes
 - ✔️ Package version updates
   - ✔️ Patch versions with fixes
-  - ✔️ Minor versions with new functionality, but no breaking changes
+  - ✔️ Minor versions with new functionality, verified to include no breaking changes
```

Comment:
I've still seen plenty of cases where people think they have verified there are no breaking changes when there are. I think people have a very limited idea of what a "breaking change" is. Perhaps we should talk more about "behaviour changes"?

Though I'd personally remove this line entirely, because people often use this as a green light for backporting x.y releases, citing "semantic versioning", even though x.y releases can be quite significant for a lot of projects.

Reply:
> I think people have a very limited idea of what a "breaking change" is. Perhaps we should talk more about "behaviour changes"?

I'm up for either term. I think the current failure is that we don't really define it.

> Though I'd personally remove this line entirely, ...

I agree, we're kinda asking them to prove a negative.
Whether to drop the line or not I believe is a `label:significant` change however, as it would mean the release channels to stray further from Fedora and more towards Debian

Reply:
Instead of version, we just say "update" to disconnect it from semantic versioning.